functions/helpers/customChannels/customChannelToFlex.private.ts

```diff
@@ -0,0 +1,298 @@
+import { Context } from '@twilio-labs/serverless-runtime-types/types';
+
+/**
+ * Looks in Sync Service for the userChannelMap named after uniqueUserName
+ */
+const retrieveChannelFromUserChannelMap = async (
+  context: Context,
+  {
+    syncServiceSid,
+    uniqueUserName,
+  }: {
+    syncServiceSid: string;
+    uniqueUserName: string;
+  },
+): Promise<string | undefined> => {
+  try {
+    const userChannelMap = await context
+      .getTwilioClient()
+      .sync.services(syncServiceSid)
+      .documents(uniqueUserName)
+      .fetch();
+
+    console.log('userChannelMap: ', userChannelMap.data);
+
+    return userChannelMap.data.activeChannelSid;
+  } catch (err) {
+    return undefined;
+  }
+};
+
+/**
+ * Creates a user channel map in Sync Service to contain the sid of the new channel assigned for a user
+ */
+export const createUserChannelMap = async (
+  context: Context,
+  {
+    syncServiceSid,
+    uniqueUserName,
+    channelSid,
+  }: {
+    syncServiceSid: string;
+    uniqueUserName: string;
+    channelSid: string;
+  },
+) => {
+  const userChannelMap = await context
+    .getTwilioClient()
+    .sync.services(syncServiceSid)
+    .documents.create({
+      data: { activeChannelSid: channelSid },
+      uniqueName: uniqueUserName,
+      ttl: 259200, // auto removed after 3 days
+    });
+
+  console.log('userChannelMap was created from scratch and its value is: ', userChannelMap.data);
+};
+
+/**
+ * Sends a new message to the provided chat channel
+ */
+export const sendChatMessage = async (
+  context: Context,
+  {
+    chatServiceSid,
+    channelSid,
+    from,
+    messageText,
+  }: {
+    chatServiceSid: string;
+    channelSid: string;
+    from: string;
+    messageText: string;
+  },
+) => {
+  const message = await context
+    .getTwilioClient()
+    .chat.services(chatServiceSid)
+    .channels(channelSid)
+    .messages.create({
+      body: messageText,
+      from,
+      xTwilioWebhookEnabled: 'true',
+    });
+
+  console.log('Message sent: ', messageText);
+
+  return message;
+};
+
+export const removeChatChannel = async (
+  context: Context,
+  {
+    chatServiceSid,
+    channelSid,
+  }: {
+    chatServiceSid: string;
+    channelSid: string;
+  },
+) =>
+  context
+    .getTwilioClient()
+    .chat.services(chatServiceSid)
+    .channels(channelSid)
+    .remove();
+
+export type AseloCustomChannels = 'twitter' | 'instagram';
```

Comment:
```suggestion
export enum AseloCustomChannel {
 Twitter = 'twitter',
 Instagram = 'instagram'
}
```

I know enums and literal unions do the same thing, but could we perhaps use enums where possible? It's just more obvious when you refer to an enum value in code that it's something predefined & from a restricted set of values, whereas a literal might just be a random string, or it might be from a restricted union, you can't immediately tell

Reply:
Yeah I may be relying too much on the inference of VSCode and that's why using unions is clear to me (hovering on a type hints the union, when type is inferred autocompletion offers the possible values, etc). 
Happy to change to enum here, however I have read in several places ([like here](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)) that enums are more of a use-case than the default to-go, so I'm not so convinced with the "where possible" part :P

Reply:
I've implemented the changes here https://github.com/techmatters/serverless/pull/206. Feel free to merge against the base PR if you think is good!